tuna/session_mixin.py

```diff
@@ -25,29 +25,35 @@
 #
 ###############################################################################
 """Session table and its associate functionality"""
+
+import logging
+import argparse
 from sqlalchemy import Column, Integer, String
 from sqlalchemy.exc import IntegrityError
-
 from tuna.dbBase.sql_alchemy import DbSession
 from tuna.utils.logger import setup_logger
 
-LOGGER = setup_logger('session')
+LOGGER: logging.Logger = setup_logger('session')
 
 
 class SessionMixin():
   """Session Mixin to provide interface for the session table"""
   #pylint: disable=too-few-public-methods
+  #pylint: disable=too-many-instance-attributes
```

Comment:
is this needed? Have you tried removing it? doesnt seem like that many instance attributes

Reply:
removed